components/omega/doc/userGuide/OmegaBuild.md

```diff
@@ -15,19 +24,61 @@ The standard practice is for a user to create a separate directory where
 the build should take place and the commands below should be launched from
 that directory.
 
-To perform a standalone build, you need to execute the cmake command with
-the required CMake and Omega parameters. For example, you can specify the
-CC parameter as the C++ compiler and enable the ctest option to guide the
-Omega build process. The following example demonstrates building
-a standalone Omega with a test suite.
+To utilize a compiler name that's defined in the E3SM machine configuration
+file (`config_machines.xml`), employ the `OMEGA_CIME_COMPILER` CMake variable,
+as illustrated below:
+
```

Comment:
On some machines (eg pm), you'll also need to specify the machine via OMEGA_CIME_MACHINE. That's shown in an example below, but might want to mention it here too if people don't read that far.

Reply:
added the note of using OMEGA_CIME_MACHINE here.